document/CoreFormat.mdk

```diff
@@ -753,9 +787,9 @@ a `formattedMessage` object (&sect;[#formattedMessage]) that can be used to cons
 a formatted message that describes the result.
 
 If the `formattedMessage` property is present on a `result`, the `fullMessage` property (&sect;[#result-fullMessage])
-shall not be present.
+shall be absent.
```

Comment:
Unrelated change. Everywhere else in the spec I say "shall be absent". These were the only two places I said "shall not be present."